LayoutTests/svg/text/font-small-enlarged-minimum-larger-expected.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```

Comment:
Why are you replacing the expectation of this test? (That's precisely the one failing in the CI....)
The test LayoutTests/svg/text/font-small-enlarged-minimum-larger.svg defines a SVG document, with a local coordinate system 0x0 - 10x10, due to viewBox="0 0 10 10" and an on-screen size of 400px x 400px. A global minimum font size of 80px is set by script.

The <text> element specifies a 'font-size' of '1.0' which is up-scaled by scaling factor 40, to match the on-screen font-size that you'd need to use if you didn't scale by factor 40 due the viewBox induced transform --> 'font-size: 40px' is requested, but 80px is the minimum available. The test then exercises that the minimum font size setting (80px > 40px requested) doesn't break the up-scaling logic for SVG.

So... why are you replacing this with a HTML version, that doesn't exercise the code path at all (no minimum font-size override, but rather 2x hidpi device scale factor?)

Reply:
If I keep the test as original, it fails on all platform with 0.01% difference and if force "font-size: 80.0" then it is only fails on iOS-wk2 and it seems to show that iOS 2 is applying device scaling factor because iOS is 3x rather than 2x like other. Appreciate your guidance for it. I am happy to modify this and have tried in the past runs.

Reply:
I found the problem with iOS-wk2 test case, it was svg.css UA stylesheet restricting SVG Text zoom on iOS platform as a result when I updated test expectation, it was failing on iOS-wk2 only. I have now added separate expectations and reverted to my old patch.

Reply:
Another update - I was going through Blink commit to find another fix in the same code to deal with low-dpi screens and I added it in my fix as well.

Reply:
Will check.

Reply:
> I found the problem with iOS-wk2 test case, it was svg.css UA stylesheet restricting SVG Text zoom on iOS platform as a result when I updated test expectation, it was failing on iOS-wk2 only. I have now added separate expectations and reverted to my old patch.

When I read this, I guessed you can leave the font-size-text-small-enlarged... testcase as it is. Since you did still revert he inimum font size setting, I wonder the testcase did not need iOS specific results before?

I'm confused.